### PR TITLE
Unit test setup

### PR DESCRIPTION
**Pull request by BRHoward**
Set up testing framework and added some example placeholder tests

### CODE REVIEW

File: client/TwitterWall.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+(function() {
```
Comment (susanthomson) [inline client/TwitterWall.js]: I think this file has been renamed, no need to resurrect it here

File: server.js [hunk 2]
```diff
@@ -1,11 +1,19 @@
 var server = require("./server/server");
 var oAuthGoogle = require("./server/oauth-google");
+var Twitter = require("twitter");
 
 var port = process.env.PORT || 8080;
 var oauthClientId = "627385202945-oqedl0onib41h39quc15pufqgqp8j8cu.apps.googleusercontent.com";
 var oauthSecret =  process.env.TWEET_WALL_OAUTH_SECRET;
 
+var twitterClient = new Twitter({
```
Comment (susanthomson) [inline server.js:9]: Good call on pulling this out

File: client/TwitterWall.js (hunk 1 above)
Comment (stephen-lt-sl) [inline client/TwitterWall.js]: This can be removed from the branch

File: client/js/controllers/MainController.js [hunk 3]
```diff
@@ -1,12 +1,11 @@
-angular.module("TwitterWallApp").controller("MainController", function($scope, $http) {
+angular.module("TwitterWallApp").controller("MainController", ["$scope", "$http", function($scope, $http) {
```
Comment (stephen-lt-sl) [inline client/js/controllers/MainController.js]: As can this

**Review by stephen-lt-sl — COMMENTED** [review]
Looks good, aside from the changes made to the client files.

**Review by stephen-lt-sl — APPROVED** [review]

**Pull request merged**